composer.json

```diff
@@ -16,33 +16,30 @@
         }
     ],
     "require": {
-        "php": "~5.5|~7.0",
-        "sonata-project/admin-bundle": "~3.0",
-        "sonata-project/block-bundle": "^2.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/property-access": "~2.2|~3.0",
-        "doctrine/phpcr-odm": "~1.1",
-        "doctrine/phpcr-bundle": "~1.1",
-        "symfony-cmf/tree-browser-bundle": "~2.0",
-        "symfony-cmf/resource-rest-bundle": "~1.0"
+        "php": "^5.5 || ^7.0",
+        "sonata-project/admin-bundle": "3.x-dev@dev",
+        "symfony/framework-bundle": "^2.3 || ^3.0",
+        "symfony/property-access": "^2.2 || ^3.0",
+        "doctrine/phpcr-odm": "^1.1",
+        "doctrine/phpcr-bundle": "^1.1",
+        "symfony-cmf/tree-browser-bundle": "^2.0@dev",
```

Comment:
I agree that `@dev` is cleaner than the minimum-stability. Please note that the same rules apply to the tags as to the min stability (both are root-only).

In the CMF, I switched to min stability because there were a lot of dependencies that we don't really depend on that needed a dev stability (because of deps of deps).